frontend/lib/src/components/widgets/FileUploader/FileDropzoneInstructions.tsx

```diff
@@ -52,7 +52,7 @@ const FileDropzoneInstructions = ({
         {`Limit ${getSizeDisplay(maxSizeBytes, FileSize.Byte, 0)} per file`}
         {acceptedExtensions.length
```

Comment:
I think we should rename `acceptedExtensions` everywhere we use it in our code base to better reflect that this can be an extension or mime type. E.g. my just `acceptedFileTypes`. Otherwise, this can be a bit confusing. Also, maybe extend the comment in `FileUploader.proto` to reflect that it also can contain mime types + mime-types with wildcards.